### PR TITLE
[CI] Add pipeline files for jobs in Public Jenkins bldenv folder.

### PR DESCRIPTION
**Pull request by liushilongbuaa**
1. build docker-sonic-mgmt image
2. build sonic-slave-*[-azdevops] image.
3. cleanup dpkg cache which exists more than 30 days.

### CODE REVIEW

**Comment by liushilongbuaa** [comment]
@xumia @qiluo-msft please review.

File: azurepipelines/docker-sonic-mgmt.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Starter pipeline
```
Comment (lguohan) [inline azurepipelines/docker-sonic-mgmt.yml:1]: sonic-mgmt and sonic-slave goes to buildimage repo.

File: azurepipelines/dpkg-cache-cleanup.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+jobs:
+- job: Build
+  pool: sonicbld
+  timeoutInMinutes: 5
+  steps:
+  - checkout: none
+  - script: |
+      set -xe
+      sudo find /nfs/dpkg_cache/ -name *.tgz -mtime +30 -type f -delete
```
Comment (lguohan) [inline azurepipelines/dpkg-cache-cleanup.yml:25]: this job can also go to build image repo

File: azurepipelines/docker-sonic-mgmt.yml (hunk 1 above)
Comment (lguohan) [reply azurepipelines/docker-sonic-mgmt.yml:1]: will review once move to buildimage repo

**Pull request closed without merging**